react/__tests__/Condition.test.tsx

```diff
@@ -185,3 +186,38 @@ test('Switches from rendering the else component to a matched condition componen
   expect(queryByText(/Hooray!/)).toBeTruthy()
   expect(queryByText(/Oh no!/)).toBeFalsy()
 })
+
+test('Prevents Condition from rendering if context match is falsy', () => {
```

Comment:
I think the test is more about the behaviour than checking the internal structure

```suggestion
test('Prevents Condition from rendering at same time of Else', () => {
```